# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationParent.h
# Commit: 33642b1c547b
# Full Hash: 33642b1c547b01c7a59417c8c2fe2a5ca33847c1
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 9: Implement alertclickcallback behavior r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227694
# ==============================================================================

diff -r 56ab49acff35 -r 33642b1c547b dom/notification/NotificationParent.h
--- a/dom/notification/NotificationParent.h	Thu Nov 07 17:32:06 2024 +0000
+++ b/dom/notification/NotificationParent.h	Thu Nov 07 17:32:07 2024 +0000
@@ -46,6 +46,7 @@
   ~NotificationParent() = default;
 
   nsresult Show();
+  nsresult FireClickEvent();
 
   void GetAlertName(nsAString& aRetval) {
     if (mAlertName.IsEmpty()) {